Makefile

```diff
@@ -12,20 +7,23 @@ export TAG = $(shell grep -e ^TAG ${ENV_FILE} | awk -F'[=]' '{gsub(/ /,""); prin
 export GIT_HASH = $(shell git rev-parse --short HEAD)
 export COMPOSE_FILE_ARGS = -f docker-compose.yml -f docker-compose.dev.yml
 
+DETACHED: ## Run docker in "detached" mode (e.g. `make DETACHED start`, etc.)
```

Comment:
I'm not sure what the advantage of running `make DETACHED start` is over running simply `docker compose -d up`. If the idea is that people can't be bothered to learn the docker cli commands and arguments, I get that, up to a point. But eventually we start to emulate more and more bits of the docker cli arguments in the makefile and it no longer feels simpler than just running `docker compose` commands "by hand".
That said, I see no reason not to approve this change as it is probably a good default for most users.